config/samples/kustomization.yaml

```diff
@@ -1,5 +1,5 @@
 ## Append samples of your project ##
 resources:
 - aiconnect_v1alpha1_ansibleaiconnect.yaml
-- mcpserver_v1alpha1_ansiblemcpconnect.yaml
+- mcpconnect_v1alpha1_ansiblemcpconnect.yaml
```

Comment:
This change is not needed, but I want to rename the file name to be consistent with `aiconnect_v1alpha1_ansibleaiconnect.yaml` in the same directory.